src/main/java/org/folio/circulation/resources/RequestQueueResource.java

```diff
@@ -1,24 +1,43 @@
 package org.folio.circulation.resources;
 
+import static java.util.concurrent.CompletableFuture.completedFuture;
+
+import java.util.concurrent.CompletableFuture;
+
 import io.vertx.core.http.HttpClient;
 import io.vertx.ext.web.Router;
 import io.vertx.ext.web.RoutingContext;
 import org.folio.circulation.domain.MultipleRecords;
 import org.folio.circulation.domain.RequestQueueRepository;
+import org.folio.circulation.domain.RequestRepository;
 import org.folio.circulation.domain.RequestRepresentation;
+import org.folio.circulation.domain.ServicePointRepository;
+import org.folio.circulation.domain.UpdateRequestQueue;
+import org.folio.circulation.domain.reorder.ReorderQueueRequest;
+import org.folio.circulation.domain.validation.RequestQueueValidation;
+import org.folio.circulation.resources.context.ReorderRequestContext;
 import org.folio.circulation.support.Clients;
 import org.folio.circulation.support.OkJsonResponseResult;
+import org.folio.circulation.support.Result;
+import org.folio.circulation.support.RouteRegistration;
 import org.folio.circulation.support.http.server.WebContext;
 
+import io.vertx.core.json.JsonObject;
+
 public class RequestQueueResource extends Resource {
+  private static final String CIRCULATION_REQUESTS_QUEUE_URI = "/circulation/requests/queue/";
```

Comment:
Why is this defined here when it is only used in the `register` method?

Reply:
This is kind an common part of RequestQueueResource's mapping, it make sense to declare it as a constant.